arch/arm/soc/st_stm32/stm32l4/Kconfig.defconfig.stm32l496xx

```diff
@@ -0,0 +1,20 @@
+# Kconfig - ST Microelectronics STM32L496xx MCU
+#
+# Copyright (c) 2016 Open-RnD Sp. z o.o.
+# Copyright (c) 2016 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32L496XX
```

Comment:
Ok, I also missed this.

I will push an update for the L432KC aswell, since I also have a fix for pinmux on the nucleo-l432kc